tex/homology/cellular.tex

```diff
@@ -201,6 +201,7 @@ \section{Cellular chain complex}
 are called the \vocab{Betti numbers} of $X$.
 In fact, we can use this to define $\chi(X)$ for any reasonable space;
 we are happy because in the (frequent) case that $X$ is a CW complex,
+the definition coincides with the normal definition of the Euler characteristic.
```

Comment:
I just invent this line, maybe it doesn't suit your tone.

Reply:
this looks fine to me